tests/python_client/testcases/test_compaction.py

```diff
@@ -322,6 +322,10 @@ def test_compact_max_time_interval(self):
 
         sleep(ct.max_compaction_interval + 1)
 
+        # create index
+        default_index = {"index_type": "IVF_FLAT", "params": {"nlist": 128}, "metric_type": "L2"}
+        collection_w.create_index("float_vector", default_index)
+
         # verify queryNode load the compacted segments
```

Comment:
Thank you so much for the update it's great. Two little tips:
1. Replace `float_vector` with var `ct.default_float_vec_field_name`
2. No need to create a new variable `default_index`, just use `ct.default_index` directly

Reply:
> Thank you so much for the update it's great. Two little tips:
> 
> 1. Replace `float_vector` with var `ct.default_float_vec_field_name`
> 2. No need to create a new variable `default_index`, just use `ct.default_index` directly

OK!

Reply:
> Thank you so much for the update it's great. Two little tips:
> 
> 1. Replace `float_vector` with var `ct.default_float_vec_field_name`
> 2. No need to create a new variable `default_index`, just use `ct.default_index` directly

Done.Thanks for your advice. ^_^